src/openrct2/network/NetworkBase.cpp

```diff
@@ -46,7 +46,7 @@ using namespace OpenRCT2;
 // It is used for making sure only compatible builds get connected, even within
 // single OpenRCT2 version.
 
-#define NETWORK_STREAM_VERSION "1"
+#define NETWORK_STREAM_VERSION "2"
```

Comment:
@ZehMatt minor annoyance that this triggers the clang-tidy check but i guess this is what we are wanting it to trigger on.

Reply:
We can add it to the list of exceptions or we actually turn that into a constexpr.